paper-dialog-behavior.html

```diff
@@ -32,11 +27,36 @@
             <paper-button dialog-confirm>Accept</paper-button>
         </div>
     </paper-dialog-impl>
--->
 
-<script>
+`paper-dialog-common.css` provide styles for a header, content area, and an action area for buttons.
+Use the `<h2>` tag for the header and the `buttons` class for the action area. You can use the
+`paper-dialog-scrollable` element (in its own repository) if you need a scrolling content area.
+
+Use the `dialog-dismiss` and `dialog-confirm` attributes on interactive controls to close the
+dialog. If the user dismisses the dialog with `dialog-confirm`, the `closingReason` will update
+to include `confirmed: true`.
+
+### Styling
+
+The following custom properties and mixins are available for styling.
 
-  Polymer.PaperDialogBehavior = Polymer.IronOverlayBehavior.concat({
+Custom property | Description | Default
+----------------|-------------|----------
+`--paper-dialog-background-color` | Dialog background color                     | `--primary-background-color`
```

Comment:
Yes, I think it would be better to list the custom properties in the docs, rather than having to go to the source.